### PR TITLE
Add pocket definition to README

### PR DESCRIPTION
**Pull request by kabu00002**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,9 +33,12 @@ pip install -e KinFragLib
 ```
 
 ### Input
-* `.flexx` and `.hydescorer` files need to be prepared and downloaded for evey subpocket using the [SeeSAR GUI](https://www.biosolveit.de/SeeSAR). All these files need to be placed in one directoy that is named **only** by the pdb ID  of the structure (e.g., `5n1f`) and should be named according to the following scheme: `<subpocket>.flexx` (`<subpocket>.hydescorer`), `<subpocket>` needs to be replaced with the subpocket of this file. E.g. the FlexX file for the AP subpocket needs to be stored as `AP.flexx`. By default, the program will search for this directory within the `config` directory, however, this can be changed wihtin the `settings.json` file. For an example see the [`config/5n1f`](config/5n1f) directory.
+* `.flexx` and `.hydescorer` bindingpocket, and pharmacophore (i.e., subpocket) definition files need to be prepared and downloaded for evey subpocket using the [SeeSAR GUI](https://www.biosolveit.de/SeeSAR).
```
Comment (PaulaKramer) [inline README.md:36]: spelling binding pocket

File: README.md [hunk 2]
```diff
@@ -33,9 +33,12 @@ pip install -e KinFragLib
...
 * Create a JSON configuration file, such that the structure pdb id, the core subpocket, path of subpockets for fragment growing, the path to the fragment library, the path to the FlexX and HYDE executeable, and the path to the folder conatining folder (named with the PDB ID) with the `.flexx` and `.hydescorer` files is defined. A template file is given in [`config/templates/settings.json`](config/templates/settings.json), where all **required** arguments are labeled with a `TODO` (`TODO` needs to be replaced by the argument) and all *optional* arguments are set by their deafult value. [`config/5n1f/settings.json`](config/5n1f/settings.json) provides an example configuration file.
 
+**Note:** We provide a script ([`pocket_definition/pocket_definition.py`](pocket_definition)) that determines the 89 KLIFS binding pocket and subpocket-specific anchor residues which might be handy if the kinase structure of interest is not listed in [KLIFS](https://klifs.net/).
```
Comment (PaulaKramer) [inline README.md:40]: Just double checking, I thought it's 85 residues?

**Review by PaulaKramer — COMMENTED** [review]
Thanks @kabu00002! Can you mention the evaluation folder at the bottom of the readme? I think you could add it in the output section.

File: README.md (hunk 1 above)
Comment (kabu00002) [inline README.md:36]:
```suggestion
* `.flexx` and `.hydescorer` binding pocket, and pharmacophore (i.e., subpocket) definition files need to be prepared and downloaded for evey subpocket using the [SeeSAR GUI](https://www.biosolveit.de/SeeSAR).
```

File: README.md (hunk 2 above)
Comment (kabu00002) [inline README.md:40]:
```suggestion
**Note:** We provide a script ([`pocket_definition/pocket_definition.py`](pocket_definition)) that determines the 85 KLIFS binding pocket and subpocket-specific anchor residues which might be handy if the kinase structure of interest is not listed in [KLIFS](https://klifs.net/).
```

**Pull request merged**